week-1/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,20 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+var mentors = ["Daniel", "Irina", "Mimi", "Rob", "Yohannes"];
+function percent(mentorcount, studentscount) {
+  var mentorpercent = Math.round(
+    mentorcount / (mentorcount + studentscount) * 100
+  );
+  var studentspercent = Math.round(
+    studentscount / (mentorcount + studentscount) * 100
+  );
+
+  console.log("Percentage mentors: " + mentorpercent + "%");
+  console.log("Percentage students: " + studentspercent + "%");
+}
```

Comment:
Looking at the instructions, I think the idea is that you write a reusable function that calculates a percentage, given a number and a total, rather than calculating both percentages in a single function. It also says you should log the messages separately (you're logging the values inside the function which does the calculation).